antora/docs/rule_data.yml

```diff
@@ -0,0 +1 @@
+../../data/rule_data.yml
```

Comment:
This should probably go in the [attachments](https://docs.antora.org/antora/latest/attachments-directory/) directory.